items/armor/armor.json

```diff
@@ -2398,12 +2401,71 @@
 	},
 	"2310126051": {
 		"name": "CE-101 Guerilla Gorilla",
+		"description": "Heavily insulated from animal bites, enemy fire, and the histrionic diatribes of anarchists.",
 		"type": 1,
 		"slot": 0,
 		"armor_rating": 100,
 		"speed": 100,
 		"stamina_regen": 100,
 		"passive": 0
-	}	
-
-}
+	},
+	"1": {
+		"name": "SR-24 Street Scout",
+		"description": "This armor's nostalgic color scheme brings to mind beloved trappings of colonial yesteryear: concrete frontier domiciles, slate-colored civilian jumpsuits, and Space Settlers' Pie.",
+		"type": 0,
+		"slot": 2,
+		"armor_rating": 50,
+		"speed": 550,
+		"stamina_regen": 125,
+		"passive": 14
+	},
+	"23": {
+		"name": "SR-24 Street Scout",
+		"description": "This armor's nostalgic color scheme brings to mind beloved trappings of colonial yesteryear: concrete frontier domiciles, slate-colored civilian jumpsuits, and Space Settlers' Pie.",
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"3": {
+		"name": "Greatcloak of Rebar Resolve",
+		"description": "Inside the collar is embroidered: 'Wherever I may plant my shield, know that I will never yield.' and 'machine washable'.",
+		"type": 1,
+		"slot": 1,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"12": {
+		"name": "SR-18 Roadblock",
+		"description": "The crest on this armor's shoulder shows friendlies that safety is near, and foes that they have reached the end of the road.",
+		"type": 1,
+		"slot": 0,
+		"armor_rating": 100,
+		"speed": 100,
+		"stamina_regen": 100,
+		"passive": 0
+	},
+	"13": {
+		"name": "SR-18 Roadblock",
+		"description": "The crest on this armor's shoulder shows friendlies that safety is near, and foes that they have reached the end of the road.",
+		"type": 2,
+		"slot": 2,
+		"armor_rating": 150,
+		"speed": 450,
+		"stamina_regen": 50,
+		"passive": 14
+	},
+	"14": {
+		"name": "Holder of the Yellow Line",
```

Comment:
Can we add the `description` field here, even if it's just an empty string?
this could prevent certain tools breaking on missing fields

Reply:
I'll try and find the actual description if possible too